python/arcticdb/storage_fixtures/s3.py

```diff
@@ -339,7 +337,8 @@ def _start_server(self):
             self.client_cert_file = ""
         self.client_cert_dir = self.working_dir
         
-        self._p = multiprocessing.Process(
+        spawn_context = multiprocessing.get_context('spawn')
```

Comment:
Could we have a comment explaining what the spawn_context is for and how it avoids the issue?